credential/schema/vcjsonschema_test.go

```diff
@@ -254,6 +254,53 @@ func TestIsCredentialValidForJSONSchema_JsonSchemaCredential(t *testing.T) {
 		})
 	})
 
+	t.Run("2.2 The value of the credentialSchema property MUST always be set to [known json schema]", func(t *testing.T) {
```

Comment:
Can you clarify that the 2.2 refers to a specific test from teh spec test suite?

Reply:
yes